pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -62,6 +62,13 @@ rustPlatform.buildRustPackage {
     ROCKSDB_LIB_DIR = "${rocksdb}/lib";
   };
 
+  postInstall = ''
+    mkdir -p $out/etc/stalwart $out/share/web/
+    cp resources/config/spamfilter.toml $out/etc/stalwart/spamfilter.toml
+    cp -r resources/config/spamfilter $out/etc/stalwart/
+    cp resources/webadmin.zip $out/share/web/
```

Comment:
fyi: since 0.8.1 (being specific https://github.com/stalwartlabs/mail-server/commit/0fc5d40ec1442d3632a25603c50385916a695bb1) the mail-server repo no longer prodives a zip for the webadmin interface.